src/components/repo-list-item.component.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
```

Comment:
Why do we have this component if we already have `repository-list-item`?

Reply:
This is a temporary thing. I wanted to avoid cluttering `repository-list-item` with if/else to target the correct attribute. Turns out not so many things changed.

`repo-list-item` will disappear in favor of `repository-list-item`.